pom.xml

```diff
@@ -124,13 +124,18 @@ limitations under the License.
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.9.8</version>
+            <version>2.9.10.4</version>
         </dependency>
         <dependency>
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
             <version>1.1.3</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>27.0.1-jre</version>
```

Comment:
v2 async also had dependency on guava, however as `azure-cosmosdb` comes in your maven pom file first, maven will resolve guava through `azure-cosmosdb` as a transitive dependnecy.

If you want to bring in earlier version of gauva you should put this at top of your maven dependency list and explicilty exclude guava from azure-cosmosdb

Reply:
Versions are identical - but makes sense - will change the order